packages/suite/src/utils/suite/messageSystem.ts

```diff
@@ -127,20 +131,50 @@ export const validateDeviceCompatibility = (
     }
 
     const deviceFwVersion = getFwVersion(device);
+    const deviceBootloaderVersion = getBootloaderVersion(device);
+    const deviceFwRevision = getFwRevision(device);
     const deviceFwVariant = isBitcoinOnly(device) ? 'bitcoin-only' : 'regular';
     const model = getDeviceModel(device);
     const { vendor } = device.features;
 
     return deviceConditions.some(
         deviceCondition =>
             deviceCondition.model.toLowerCase() === model.toLowerCase() &&
-            deviceCondition.vendor.toLowerCase() === vendor.toLowerCase() &&
-            semver.satisfies(deviceFwVersion, createVersionRange(deviceCondition.firmware)!) &&
+            (deviceCondition.vendor.toLowerCase() === vendor.toLowerCase() ||
```

Comment:
not sure if will look better but maybe some desctructuring could improve readability?

Reply:
ec64e866ad9eeae7505faa4fbd96adb3f0404ed6